sdk/go/common/resource/plugin/plugin.go

```diff
@@ -45,6 +45,11 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
 )
 
+const (
+	PulumiDebugProvidersEnvVar = "PULUMI_DEBUG_PROVIDERS"
+	PulumiDebugLanguagesEnvVar = "PULUMI_DEBUG_LANGUAGES"
```

Comment:
Probably not in this PR, but leave a TODO that these should use the global envvar system. I'll fix it up later.

Reply:
can you add me on that so I can see it I'm not familiar